docs/Sources.md

```diff
@@ -72,6 +72,36 @@ source:
       enable: enable
 ```
 
+### Custom SQL Query Support
+
+You may also implement a custom SQL query to retreive the nodelist using  SQL syntax with the `query:` configuration parameter under the `sql:` stanza.
+
+
+#### Custom SQL Query Examples
+
+You may have a table named `nodes` which contains a boolean to indicate if the nodes should be enabled (fetched via oxidized). This can be used in the custom SQL query to avoid fetching from known impacted nodes.
+
+In your configuration, you would add the `query:` parameter and specify the SQL query. Make sure to put this within the `sql:` configuration section.
+
+```sql
+query: "SELECT * FROM nodes WHERE enabled = True"
+```
+
+Since this is an SQL query, you can also provide a more advanced query to assist in more complicated oxidized deployments. The exact deployment is up to you on how you design your database and oxidized fetchers.
+
+In this example we limit the nodes to two "POPs" of `mypop1` and `mypop2`. We also require the nodes to have the `enabled` boolean set to `True`.
+
+```sql
+query: "SELECT * FROM nodes WHERE pop ('mypop1','mypop2') AND enabled = True"
```

Comment:
I think the correct SQL syntax is:

SELECT * FROM nodes WHERE pop IN ('mypop1','mypop2') AND enabled = True

(Missing IN)

Reply:
Thanks! Good catch. Absolutely required 'IN' - thanks for the fixup.  I had originally done the example as multi-POP. Then thought to make it simpler and started to do the example as a single-POP example. Then decided to go back to multi-POP as I only wanted two query examples as not to break the flow of the `Sources.md` (probably start to split them out to `Sources-<source>.md`).
 
So this is what happens when you change your mind two times mid-query :)